### ISSUE TITLE
Invalid or unknown policy when adding a SAN whitelist to signing profile

### ISSUE THREAD

**Issue by cmrunton** [cloudflare/cfssl#627]
I've attempted to add a `name_whitelist` ([as described in cmd/cfssl.txt](https://github.com/cloudflare/cfssl/blob/master/doc/cmd/cfssl.txt)) to a signing policy and while the server starts normally, I recieve the following response from the CA when attempting to issue a certificate via the API:

`{"code":7400,"message":"{\"success\":false,\"result\":null,\"errors\":[{\"code\":5200,\"message\":\"Invalid or unknown policy\"}],\"messages\":[]}\n"}`

Without the `name_whitelist` attribute in the signing profile I can issue certificates normally. 

Below is the config.json used to start the CA server. Note that I had to double-escape the `*` in the whitelist regex in order to get the server to start, otherwise it throws a `Failed to load config file: {"code":5200,"message":"failed to unmarshal configuration: invalid character '*' in string escape code"}` error. 

```json
{
  "signing": {
    "default": {
      "usages": ["digital signature"],
      "expiry": "8000h"
    },
    "profiles": {
      "server": {
        "usages": ["server auth","key encipherment","signing"],
        "expiry": "2160h",
        "name_whitelist":"([^\\*]+).(example.*).(it|us|com)"
      }
    }
  }
}
```

**Comment by clly** [comment]
I actually ran into this and create a pull request for it. https://github.com/cloudflare/cfssl/pull/644

**Comment by kisom** [comment]
Closed by #644.
